hdfs-commons/src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -34,7 +34,8 @@
   private static final int DEFAULT_JOURNAL_NODE_COUNT = 3;
   private static final int DEFAULT_FAILOVER_TIMEOUT = 31449600;
```

Comment:
not you, but default_failover_timeout should be renamed default_failover_timeout_sec to be consistent with other code